data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconValidatorsHandler.java

```diff
@@ -69,7 +74,13 @@ public BeaconValidatorsHandler(
         @OpenApiParam(
             name = ACTIVE,
             description =
-                "If specified, return only validators which are active in the specified epoch")
+                "If specified, return only validators which are active in the specified epoch."),
+        @OpenApiParam(
+            name = PAGE_SIZE,
+            description = "If specified, return only this many results."),
```

Comment:
should comment here that the default pagination is X (ideally reference the variable).

Reply:
done